OpenOversight/tests/test_utils.py

```diff
@@ -179,28 +187,25 @@ def test_compute_hash(mockdata):
     assert hash_result == expected_hash
 
 
-def test_s3_upload_png(mockdata, test_png_BytesIO):
-    mocked_connection = Mock()
-    mocked_resource = Mock()
-    with patch("boto3.client", Mock(return_value=mocked_connection)):
-        with patch("boto3.resource", Mock(return_value=mocked_resource)):
-            OpenOversight.app.utils.upload_obj_to_s3(test_png_BytesIO, "test_cop1.png")
-
-    assert (
-        mocked_connection.method_calls[0][2]["ExtraArgs"]["ContentType"] == "image/png"
-    )
-
+@pytest.mark.parametrize(
+    "extension,mime_type",
+    [
+        ("jpeg", "image/jpeg"),
+        ("png", "image/png"),
+        ("gif", "image/gif"),
+        ("webp", "image/webp"),
```

Comment:
Maybe we should add some upper case test cases here as well?

Reply:
In this case, we're detecting the file type solely using the file header (not extension) and the extension is only used to generate the test image byte data